app/scripts/directives/podDonut.js

```diff
@@ -49,7 +49,7 @@ angular.module('openshiftConsole')
           type: "donut",
           bindto: '#' + $scope.chartId,
           donut: {
-            expand: false,
+            expand: true,
```

Comment:
yeah, I assumed we wanted to switch behaviors (remove opacity change and add expand). Maybe we can disable the expand for empty donuts?